packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -202,6 +203,23 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
     data.avatarUrl = null;
     data.avatar = null;
   }
+  if (input.completedOnboarding) {
+    const userTeams = await prisma.user.findFirst({
+      where: {
+        id: user.id,
+      },
+      select: {
+        teams: true,
```

Comment:
how about just getting the team id as we don't need all the columns?

teams : {
 id: true
}

Reply:
Absolutely